README.md

```diff
@@ -59,13 +59,17 @@ kedro viz
 ```
 
 This command will run kedro_viz.server on `http://127.0.0.1:4141/` which cannot be accessed from another machine.
-If you are using a remote server machine or a docker container, run the following command instead.
 
-```bash
-kedro viz --host 0.0.0.0
-```
-
-You can change the port with ```--port``` option if needed.
+Kedro-Viz has a number of options to customise running the visualisation. Here is a list of CLI arguments supported:
+| CLI command              | Description                                                                                |
+| ------------------------ | ------------------------------------------------------------------------------------------ |
+| `--host`                 | Host that viz will listen to. Defaults to 127.0.0.1.                                       |
+| `--port`                 | TCP port that viz will listen to. Defaults to 4141.                                        |
+| `--browser/--no-browser` | Whether to open viz interface in the default browser or not.                               |
+| `--load-file`            | Path to load the pipeline JSON file                                                        |
+| `--save-file`            | Path to save the pipeline JSON file                                                        |
+| `--pipeline`             | Name of the modular pipeline to visualise. If not set, the default pipeline is visualised. |
```

Comment:
Could you hyperlink `modular pipeline` to this section in the documentation: https://kedro.readthedocs.io/en/latest/04_user_guide/06_pipelines.html#modular-pipelines